source/particle/output/ascii.cc

```diff
@@ -42,9 +42,15 @@ namespace aspect
         const std::string output_file_prefix = "particle-" + Utilities::int_to_string (file_index, 5);
         const std::string output_path_prefix = this->get_output_directory() + output_file_prefix;
         const std::string full_filename = output_path_prefix + "." + Utilities::int_to_string(Utilities::MPI::this_mpi_process(this->get_mpi_communicator()), 4) + ".txt";
+
         std::ofstream output (full_filename.c_str());
-        if (!output)
-          std::cout << "ERROR: proc " << Utilities::MPI::this_mpi_process(this->get_mpi_communicator()) << " could not create " << full_filename << std::endl;
+
+        AssertThrow (output,
+                     ExcMessage (std::string("Couldn't open ascii particle output file <"
```

Comment:
i've change this in other places to say "Could not" to be more formal. Could you make the same change here?